src/components/tabs/tab/tab.component.tsx

```diff
@@ -1,17 +1,62 @@
 import React, { useCallback, useEffect, useState } from "react";
-import PropTypes from "prop-types";
-import styledSystemPropTypes from "@styled-system/prop-types";
+import { PaddingProps } from "styled-system";
 import StyledTab from "./tab.style";
 import tagComponent from "../../../__internal__/utils/helpers/tags/tags";
-import { filterStyledSystemPaddingProps } from "../../../style/utils";
 
-const TabContext = React.createContext({});
+export interface TabContextProps {
+  setError?: (childId: string, error?: boolean | string) => void;
+  setWarning?: (childId: string, warning?: boolean | string) => void;
+  setInfo?: (childId: string, info?: boolean | string) => void;
+}
 
-const paddingPropTypes = filterStyledSystemPaddingProps(
-  styledSystemPropTypes.space
-);
+export interface TabProps extends PaddingProps {
+  title?: string;
+  /** A unique ID to identify this specific tab. */
+  tabId: string;
+  className?: string;
```

Comment:
```suggestion
  /** @ignore @private */
  className?: string;
```

Reply:
I missed this because it was only in the proptypes, not the previous `.d.ts` file. Thanks.